napari/_qt/layer_controls/_tests/test_qt_labels_layer.py

```diff
@@ -63,3 +63,22 @@ def test_changing_colormap_updates_colorbox(qtbot):
         color_box.color,
         np.round(np.asarray(layer._selected_color) * 255 * layer.opacity),
     )
+
+
+def test_layercontrols_checkboxes(qtbot):
```

Comment:
Optional: slight preference for splitting this up into multiple tests and refactoring the setup code to be shared and maybe a fixture. But this is also good enough for me right now too.

Reply:
Thanks for this comment! With the help of ChatGPT I learned about fixtures and arguments and implemented this. I hope it's correct though 😅